app/src/Navigators/Application.tsx

```diff
@@ -103,15 +105,19 @@ const ApplicationNavigator = () => {
           },
           dark: false,
         }
+
   const safeAreaViewStyle = [
-    { backgroundColor: navigationTheme.colors.background, flex: 1 },
+    {
+      paddingTop: insets.top,
+      backgroundColor: navigationTheme.colors.background,
+      flex: 1,
+    },
   ]
 
   return (
-    <SafeAreaView style={safeAreaViewStyle}>
+    <View style={safeAreaViewStyle}>
```

Comment:
SafeAreaView was causing white padding at the bottom of the screen in iOS. This was resolved by switching to the hook useSafeAreaInsets. This is recommended from version 6.x. More info here:  https://reactnavigation.org/docs/handling-safe-area/